### ISSUE TITLE
Can't make threads on /vt/

### ISSUE THREAD

**Issue by nambona890** [K1rakishou/Kuroba-Experimental#869]
For some reason, and this doesn't happen on any other board that I've tested, trying to make a thread on /vt/ results in this error message popping up:
![image](https://user-images.githubusercontent.com/38313501/218993771-3e4bf135-6bd5-46e4-85ba-2bc52bf86e01.png)
I can confirm that it has nothing to do with banned text, banned images, nor bans/rangebans. Making replies to existing threads works fine. On my PC with 4chanX I'm able to post just fine, even on mobile data when I hotspot it.

**Comment by K1rakishou** [comment]
And what happens if you change user-agent to something random?

**Comment by nambona890** [comment]
> And what happens if you change user-agent to something random?

The CloudFlare check hangs.

**Comment by 1sad** [comment]
just tested, same issue. tried "KurobaEx/v1.2.3.4” and "KurobaEx/v1.3.23” useragent (cloudflare works), as well as just putting webview agent in (cloudflare hangs)

interestingly, if I put even parts of the real user agent in, like just "Mozilla/5.0" or "AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/109.0.5414.117 Mobile Safari/537.36" cloudflare hangs too.

![IMG_20230215_164612](https://user-images.githubusercontent.com/23120024/219062313-b9bb04bc-987b-4921-b5af-bb2113c71fec.jpg)

**Comment by K1rakishou** [comment]
Can't reproduce. Just created a thread successfully - https://boards.4chan.org/vt/thread/43178758

**Comment by K1rakishou** [comment]
Oh wait, for some reason postNo is 0 on both images. And in the code in case when either postNo or threadNo is 0 it's assumed that we failed to post. But does it actually create a thread on 4chan, did you check that? It may say that it failed but it seems that the server created the thread (since there was no error message in the response). The only problem is that postNo is 0 which I can fix by assigning it to threadNo (they should be the same when creating threads). This seems like a 4chan bug. Probably need to report it (even though they never fix them).

**Comment by nambona890** [comment]
> But does it actually create a thread on 4chan, did you check that?

No, it doesn't. At least not for me. What useragent are you using?

> This seems like a 4chan bug.

On some devices on some boards, for whatever reason making a new thread just gives you a "Post successful!" instead of the "(image) uploaded!", and redirects you to a random thread without posting anything. I theorize it's to do with a web app called 'ecker which let you post truly anonymously, but it's now only accessible by sending a dick pic to the owner, granted you know what altchan to find him on.

**Comment by K1rakishou** [comment]
>No, it doesn't. At least not for me. 

Weird.

>What useragent are you using?

The default one (the one WebView is using). For some reason I can now post using it.

>for whatever reason making a new thread just gives you a "Post successful!" instead of the "(image) uploaded!"

Yeah, this is a shadownban they use to fight the bots.

But this issue here is something different. It seems like a bug on their side.

**Comment by 1sad** [comment]
i just tried again on /vt/, and it just worked???... (useragent KurobaEx/v1.2.3.4 , default webview one still broken)
must be weird 4chan things.
